app/controllers/shipit/api/tasks_controller.rb

```diff
@@ -3,14 +3,14 @@ module Shipit
   module Api
     class TasksController < BaseController
       require_permission :read, :stack
-      require_permission :deploy, :stack, only: :trigger
+      require_permission :deploy, :stack, only: %i(trigger abort)
```

Comment:
Just wanted to make sure I understand this line, does this mean that permissions are checked only before the abort method is run?

Reply:
So the list of available API permissions are as follows (from `api_client.rb`):
```
      read:stack
      write:stack
      deploy:stack
      lock:stack
      read:hook
      write:hook
```

This line is saying that you need `deploy:stack` permission _only_ if you are hitting the `trigger` or `abort` actions. For all the other actions in this controller, you only need the `read:stack` permission.